screens/Market/MarketScreen.jsx

```diff
@@ -80,9 +95,12 @@ const MarketScreen = ({ navigation }) => {
           key={item => item.id}
           keyExtractor={item => item.id}
           renderItem={({ item }) => {
-            return <Item item={item} navigation={navigation} />;
+            return  isLoading ? <MarketSkeletonItem/>
+              : <Item item={item} navigation={navigation} />
+              ;
```

Comment:
I saw this callback in 3 places. Create a callback and reuse

Reply:
I am looking at several solutions for this situation. Tomorrow morning I will test the most compact and optimal way to solve !